src/uint/div.rs

```diff
@@ -129,32 +238,133 @@ impl<const LIMBS: usize> Uint<LIMBS> {
     /// When used with a fixed `rhs`, this function is constant-time with respect
     /// to `self`.
     pub const fn rem_wide_vartime(lower_upper: (Self, Self), rhs: &NonZero<Self>) -> Self {
```

Comment:
Could this be reduced to `div_rem_vartime()` by widening `rhs` to double width, and then halving the result? So that you don't have to spell out the whole algorithm again.

Reply:
I don't think that can be done without extra trait bounds or const evaluation support.